src/main/telemetry/smartport.c

```diff
@@ -130,6 +132,7 @@ enum
     FSSP_DATAID_ADJFUNC    = 0x5110 , // custom
     FSSP_DATAID_ADJVALUE   = 0x5111 , // custom
     FSSP_DATAID_CAP_USED   = 0x5250 ,
+    FSSP_DATAID_GMODE      = 0x5450 , //custom
```

Comment:
GMODE is not very descriptive. Maybe GOV_MODE

Reply:
Done